src/_nebari/stages/kubernetes_services/template/rook-ceph.tf

```diff
@@ -45,6 +45,13 @@ resource "helm_release" "rook-ceph" {
       },
       csi = {
         enableRbdDriver = false, # necessary to provision block storage, but saves some cpu and memory if not needed
+        provisionerReplicas : 1, # default is 2 on different nodes
+        pluginTolerations = [
```

Comment:
runs csi-driver on all nodes, even those with NoSchedule taints.  Doesn't run on nodes with NoExecute taints.  This is what the nebari-prometheus-node-exporter daemonset does so I copied it here.